NEWS

```diff
@@ -2,6 +2,166 @@ News
 ====
 
 
+Version 3.1.0 rc1
```

Comment:
Just for completeness, I assume skipping a beta release is intentional here? If yes, I'm fine with no beta.